g2p/mappings/langs/generated/config.yaml

```diff
@@ -252,4 +243,17 @@ mappings:
     out_lang: eng-ipa
     prevent_feeding: true
     reverse: false
-    rule_ordering: as-written
\ No newline at end of file
+    rule_ordering: as-written
+  - authors:
+      - Generated 2021-05-11 08:47:29.486775
+    case_sensitive: false
+    display_name: Mohawk IPA to English IPA
+    escape_special: false
+    in_lang: moh-ipa
+    language_name: moh-ipa
+    mapping: moh-ipa_to_eng-ipa.json
+    norm_form: NFC
```

Comment:
Hm, yes, this is interesting - I want to use NFC for TTS and all the keyboards use NFC. This shouldn't be a problem since we fix the arpabet transducer to NFD though right and it should normalize to NFD after eng-ipa.

Reply:
As long as we test it and it works, it's a fine solution.